README.md

```diff
@@ -1,6 +1,12 @@
 # Organization workflows app
 
-This GitHub app allows you to centrally manage and run multiple [GitHub Actions](https://github.com/features/actions) workflows across multiple repositories. Currently this is a limitation in GitHub Actions, as it only allows you to configure and manage Actions workflows on a repository level. This app helps you - for example - to centrally define central workflows for linting, compliance checks, and more.
+This GitHub app allows you to run [GitHub Actions](https://github.com/features/actions) workflows across multiple repositories, which is [not yet natively suppoprted](https://github.com/github/roadmap/issues/52). This app helps you - for example - to create a single workflow definition that is used for linting, compliance checks, and more.
+
+This app may not be right for you if:
+* You require production quality. This app is considered expeirmental and runs on a free Heroku dyno
```

Comment:
There is a typo at "expeirmental"

Reply:
This line can be removed altogether. All resources are hosted on scalable Azure instances now. 👍🏽

Reply:
Removed!